OverlayPlugin.Core/EventSources/FFXIVClientStructsEventSource.cs

```diff
@@ -0,0 +1,204 @@
+﻿using System.Collections.Generic;
+using System.Windows.Forms;
+using Newtonsoft.Json.Linq;
+using RainbowMage.OverlayPlugin.MemoryProcessors;
+using RainbowMage.OverlayPlugin.MemoryProcessors.AtkGui.FFXIVClientStructs;
+using RainbowMage.OverlayPlugin.MemoryProcessors.AtkStage;
+
+namespace RainbowMage.OverlayPlugin.EventSources
+{
+    using Utilities = MemoryProcessors.AtkStage.FFXIVClientStructs.Utilities;
+    public class SortedPartyList
+    {
+        public string PartyType;
+        public int MemberCount;
+        public int TrustCount;
+        public byte PetCount;
+        public byte ChocoboCount;
+        public List<Entry> Entries = new List<Entry>();
+
+        public struct Entry
+        {
+            public int Index;
+            public string Name;
+            public EntryType Type;
+            public enum EntryType
+            {
+                Party = 0,
+                Trust = 1,
+            }
+        }
+    }
+
+    public class FFXIVClientStructsEventSource : EventSourceBase
+    {
+        private IAtkStageMemory atkStageMemory;
+        private FFXIVMemory memory;
+
+        public BuiltinEventConfig Config { get; set; }
+
+        public FFXIVClientStructsEventSource(TinyIoCContainer container) : base(container)
+        {
+            atkStageMemory = container.Resolve<IAtkStageMemory>();
+            memory = container.Resolve<FFXIVMemory>();
+
+            RegisterEventHandler("getFFXIVCSAddon", (msg) =>
```

Comment:
I don't know that it should be considered "debug" since there are potential legitimate use cases for this (e.g. `_Exp` addon maps to `AddonExp` which could be useful to track exp gain rate over time, and which should be really quick to retrieve).

Maybe it just needs `Slow` in the name, e.g. `getFFXIVCSAddonSlow`?

Reply:
Yeah, `Slow` seems good to me too.  Mostly just don't want people to call this constantly.